cmd/analyzer/parse_args.go

```diff
@@ -60,26 +60,52 @@ const (
 	allSubnetsDiff   = "diff_all_subnets"   // semantic diff of allSubnets analysis between two configurations
 )
 
-var supportedOutputFormats = map[string]bool{
-	JSONFormat:       true,
-	TEXTFormat:       true,
-	MDFormat:         true,
-	DRAWIOFormat:     true,
-	ARCHDRAWIOFormat: true,
-	DEBUGFormat:      true,
+var supportedOutputFormatsMap = map[string][]string{
+	JSONFormat:       {allEndpoints, allSubnets},
+	TEXTFormat:       {allEndpoints, allSubnets, singleSubnet, allEndpointsDiff, allSubnetsDiff},
+	MDFormat:         {allEndpoints, allEndpointsDiff, allSubnetsDiff},
+	DRAWIOFormat:     {allEndpoints},
+	ARCHDRAWIOFormat: {allEndpoints},
+	DEBUGFormat:      {allEndpoints},
```

Comment:
this can be changed to the map it used to be `map[string]bool`, the current map values are not used